views/country_download.erb

```diff
@@ -132,68 +132,42 @@ var max_visible_terms = 6;
 var reduced_visible_terms = 3; // best odd, so wanted term is in middle?
 var wanted_house = get_q_param('house');
 var wanted_term = get_q_param('term');
+var pageInitFunction = function(){
 
-// for each house...
+  // for each house...
+  // highlight the specific term (if there is one)
+  // collapse terms if there are many
 
-// if there are more than `max_visible_terms` terms, hide them:
-// display `reduced_visible_terms`
-//   * selected term in the middle if possible
-//   * otherwise the most recent
+  $(".country__legislature").each(function(){
+    var house_slug = $(this).data('house');
+    var $term_list_items = $('#terms-' + house_slug + ' li');
```

Comment:
Small thing, but you call them `items` here, but `elements` on the next line. Would make more sense to me to standardise. eg: `$term_list_items` and `$wanted_list_item`